pkgs/development/ocaml-modules/semver/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub, buildOasisPackage }:
+
+buildOasisPackage rec {
```

Comment:
Sounds overkill. There is no need for `oasis` here.

Reply:
So I'll revert to mkDerivation?

Reply:
Commit f84ac1e912a6f408060162c9ef88a3ffbe031839 is nearly good: it misses a `meta.platforms` and `ocaml${ocaml.version}` as part of the `pname`.

Reply:
Should be OK now then